mats_schedule/handlers/mats_schedule.py

```diff
@@ -90,13 +80,19 @@ def lambda_handler(event: Event, context: Context):
             csv_path,
             convert_options=csv.ConvertOptions(column_types=ScheduleSchema),
         )
+        data = table.to_pandas()
+        data["pointing_altitudes"] = data.apply(
+            lambda s: eval(s.pointing_altitudes),
```

Comment:
Something like `lambda s: [int(v) for v in s.pointing_altitudes[1:-1].split(",") if v]` would probably also do the trick. Let me know if you prefer this.

Reply:
splitting feels better.